home/screens/ProjectsScreen.js

```diff
@@ -19,6 +19,7 @@ import {
 } from 'react-native';
 import { withNavigationFocus, withNavigation } from 'react-navigation';
 import { connect } from 'react-redux';
+import semver from 'semver';
```

Comment:
Could you also run `yarn add semver` so we have it in package.json and yarn.lock?